<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSUchatting</class>
 <widget class="QMainWindow" name="CSUchatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>csu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="msgBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrNick">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrTicket">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>59</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加入</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
